app/models/authenticator.rb

```diff
@@ -34,9 +34,23 @@ def uses_stored_password?
       false
     end
 
+    def can_authorize_user_by_userid?
```

Comment:
@Fryguy Funny because @abellotti had `autorize_user_by_userid?` and I thought that name was odd and suggested `can_authorize_user_by_userid?`.., Naming is hard. `can_user_be_authorized_by_userid?` <== Thoughts?